MapboxCoreNavigation/SimulatedLocationManager.swift

```diff
@@ -163,8 +163,8 @@ open class SimulatedLocationManager: NavigationLocationManager {
         
         // Simulate speed based on expected segment travel time
         if let expectedSegmentTravelTimes = routeProgress?.currentLeg.expectedSegmentTravelTimes,
-            let coordinates = routeProgress?.route.coordinates,
-            let closestCoordinateOnRoute = Polyline(routeProgress!.route.coordinates!).closestCoordinate(to: newCoordinate),
+            let coordinates = routeProgress?.route.shape?.coordinates,
+            let closestCoordinateOnRoute = Polyline(coordinates).closestCoordinate(to: newCoordinate),
```

Comment:
Use `routeProgress?.route.shape` instead of copying the coordinates into a new Polyline.